File: csharp/StructuresExample.cs
using System;

struct Point {
    public int x, y;
}

class StructuresExample {
    static void Main(string[] args) {
        Point p = new Point();
        p.x = 10;
        p.y = 20;
        Console.WriteLine(p.x + " " + p.y);
    }
}

File: java/StructuresExample.java
//Java uses classes instead of structures.

class Point {
    int x, y;
    Point(int x, int y) {
        this.x = x;
        this.y = y;
    }
}

public class StructuresExample {
    public static void main(String[] args) {
        Point p = new Point(10, 20);
        System.out.println(p.x + " " + p.y);
    }
}
